js-intro.md

```diff
@@ -5,68 +5,72 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: To make some something work
 
-  Researched answer:
+  Researched answer: to perfom a task or calculate a value
```

Comment:
I would revisit this and do more research on what exactly a function is and how we utilize them